From 1043d4d3bddf2c152c84550d69678976d2ee7b2f Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 15 May 2017 11:37:04 +0000
Subject: [PATCH 0841/1076] MANTIS 0045822 MR-Component: VisionBox: PPR, Face;
 HRS: Iris

codes.idl
V1.4.2
- added extended datastatus DS_TYPES_XB*, DS_TYPES_USER_ID for biometrics



git-svn-id: svn://localhost/SelfServiceCommon/trunk@1199 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 1384a2b5..028c2146 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -6,7 +6,7 @@
 //
 //  Date:      31.03.2015
 //
-//  Version:   1.4
+//  Version:   1.4.2
 //
 //  Author:    IATA Passenger Experience Management Group: CUWG CUSS-TSG
 //
@@ -73,6 +73,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li Mantis 0040920: Conveyor3Dev: additional error codes ManualMode; SwitchedOff
 - added materna extension BAGGAGE_MANUAL_MODE, BAGGAGE_SWITCHED_OFF
 
+@date 15.05.2017
+@li V1.4.2
+@li Mantis 0045822: MR-Component: VisionBox: PPR, Face; HRS: Iris
+- added extended datastatus DS_TYPES_XB*, DS_TYPES_USER_ID for biometrics
+
 */
 
 #ifndef CODES_IDL
@@ -162,6 +167,14 @@ module datastatus
     const long DS_TYPES_SBDAEA                   = 18040; /**< AEA-SBD control language */
     const long DS_TYPES_SBDCUSS                  = 18050; /**< CUSS-SBD control language */
 
+    const long DS_TYPES_XB01                     = 19000; /**< CUSS Biometric.xsd */
+    const long DS_TYPES_XB01_SETUP               = 19010; /**< setup with CUSS Biometric.xsd */
+    const long DS_TYPES_XB01_LEFT_EYE            = 19120; /**< left eye */
+    const long DS_TYPES_XB01_RIGHT_EYE           = 19130; /**< right eye */
+    const long DS_TYPES_XB01_FACE                = 19140; /**< face */
+
+    const long DS_TYPES_USER_ID                  = 19400; /**< user identifaction */
+
     const long DS_TYPES_EPASSPORT_DG1            = 20100; /**< e-Passport format */
     const long DS_TYPES_EPASSPORT_DG2            = 20200; /**< e-Passport format */
     const long DS_TYPES_EPASSPORT_DG3            = 20300; /**< e-Passport format */
-- 
2.41.0.windows.1

